Vagrantfile

```diff
@@ -70,11 +70,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = 'pubstack'
-
-  # The url from where the 'config.vm.box' box will be fetched if it
-  # doesn't already exist on the user's system.
-  config.vm.box_url = 'http://files.vagrantup.com/precise64.box'
+  config.vm.box = 'ubuntu/trusty64'
```

Comment:
This is a much cleaner, and future-proof way to go (for example, try to visit http://files.vagrantup.com right now LOL). I don't think this option was documented (if it was available) when we first put this together 😄